packages/time-utils/tests/lib/stopwatch/common.ts

```diff
@@ -0,0 +1,3 @@
+import { promisify } from 'util';
```

Comment:
We can't use promisify with Rollup (promisify cannot be transformed to browser code)

Instead make it

```ts
const sleep = <T>(ms: number, cb?: (...args: unknown[]) => T) =>
	new Promise<T>((resolve) => {
		if (cb) setTimeout(cb, ms);
		else setTimeout(resolve, ms);
	});
```

@kyranet I think that's correct anyway? Not providing a callback returns `Promise<unknown>`, providing a callback returns whatever is returned by the callback.